package.json

```diff
@@ -11,7 +11,8 @@
     "test-prod": "NODE_ENV=production npm run unit-test && npm run e2e-test",
     "unit-test": "jest __tests__/.*.spec.js --silent",
     "e2e-test": "jest __e2e-tests__ --runInBand --silent",
-    "lint": "eslint --quiet -f codeframe src"
+    "lint": "eslint --quiet -f codeframe src",
+    "ci": "ls && npm run test-prod"
```

Comment:
I think we can drop the ls here, or is it for a purpose?

Reply:
Good catch! Will remove it, was there for testing